### ISSUE TITLE
[Feature] Integrate login with GitHub authentication option for the applicaiton

### ISSUE THREAD

**Issue by dilumdesilva** [scorelab/SCoRe-Mobile-App#17]
Since the majority of the userbase of the application are  open source contributors, organization maintainers or developers, I strongly believe having login with GitHub option would be an attractive feature for the mobile application.

**Comment by hiranthaR** [comment]
Is it necessary? I think we can fetch all the details through github APIS

**Comment by dilumdesilva** [comment]
@hiranthaR  yes true we can get most of the things by GitHub APIs but having login with GitHub option will be useful for us to track previous contributors for google summer of code, google code-in and other competitions. My idea was to map contributors using the GitHub login id and give them the ability to maintain something like socreab contributor badges sort of thing on their personal page if they logged in. 

I hope it will be useful for the contributors as well for the organization.

**Comment by hiranthaR** [comment]
Sure I'll think about it

**Comment by dhawankrish654** [comment]
Providing a GitHub authorization to the contributors and a profile for them in the mobile application is one way of encouraging their works. New developers could come to know about the organization works and we can also display projects in th android application for the user ease in place of redirecting them to GitHub URL. 
@hiranthaR 
 If this issue is still open I could readily  work upon this and make the mobile application quite useful for the organization .
